### DISCUSSION TITLE
Security of my NFS setup

### DISCUSSION THREAD

**Discussion by sjkorvin** [retronas/discussions#37] [Q&A] [answered]
Hello. I do not currently have the funds for a dedicated NAS. All of the data on my Windows PC is backed up with Backblaze Personal Backup. I wish that I could have a proper 3-2-1 backup.

I have installed RetroNAS on a Raspberry Pi 4 and I have mounted all of the drives in my PC except for my boot drive using HaneWIN NFS Server as shown in the first screenshot below.

My top-level RetroNAS directory is /mnt. In the second screenshot below, you can see that the only parameter that I set for sharing my drives is -alldirs. I'm new to NFS and sharing in general. Would you advise against what I have done? Do you think that if I'm sharing my PC's drives, then I should at least set them to be shared as read-only? I would appreciate any suggestions and advice. Thank you so much.

![Screenshot_20221127_194503_AnyDesk.png](https://user-images.githubusercontent.com/50928154/204170730-36958eb4-da71-4b41-a739-57c1f113aad8.png)

![Screenshot_20221127_194624_AnyDesk.png](https://user-images.githubusercontent.com/50928154/204170750-c555db34-e692-4b5d-a8a2-5f288f792240.png)

**Comment by sairuk** [comment, marked as answer]
I am not familiar with the software you are using nor the `-alldirs` option so cannot comment. 

I would be comfortable advising that if you are unsure about the setup and security of your data you shouldn't run it in this manner until you have enough experience to decide if this is best for you.

**Comment by sjkorvin** [comment]
That makes sense. Thank you.
